sgqlc/codegen/operation.py

```diff
@@ -262,7 +262,9 @@ def format_selection_set_inline_fragment(self, parent, type_condition,
                                              children, lines, idx):
         sel = self.selection_name(parent, '_as__%s' % type_condition, idx)
         idx += 1
-        lines.append('%s = %s.__as__(%s)' % (sel, parent, type_condition))
+        full_type_condition = str(self.schema_name) + "." + type_condition
```

Comment:
We should use `format_type_usage()` instead, I don't think we should need the `wrapper` version, but that would be a declaration error, on our side (code generation), we better use it to keep consistent.

Reply:
Just so I understand you correctly: You suggest to use another `format_type_usage()` instead, e.g. `format_type_condition()`?

Reply:
There is a method called `self.format_type_usage(typ)` that will handle the schema name prefix.